conversation/frontend/src/components/DisplayActionDropDown.tsx

```diff
@@ -217,8 +247,8 @@ export function DisplayActionDropDown({ message }: { message: Message }) {
         .map((option) => (
           <Button
             key={option.id}
-            color="primary"
-            variant="outline"
+            color={appearance.btnColor}
```

Comment:
```suggestion
            {...appearance}
```
En remplacement des 2 lignes, en renommant au préalable le champ `btnColor` en `color`